README.md

```diff
@@ -1,27 +1,58 @@
-# Prettier Python Plugin
+<p align="center">
+    :construction: Work in Progress! :construction:
+</p>
 
-![](https://i.giphy.com/media/GNvOUgBvLzVwA/giphy.webp)
+<div align="center">
+<img alt="Prettier"
+  src="https://raw.githubusercontent.com/prettier/prettier-logo/master/images/prettier-icon-light.png">
+<img alt="Python"
+  hspace="25"
+  height="210"
+  src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/0a/Python.svg/240px-Python.svg.png">
+</div>
```

Comment:
Can you use [`Python-logo-notext.svg`](https://commons.wikimedia.org/wiki/File:Python-logo-notext.svg) instead? This logo is almost ten years old.

Reply:
Oops. Thanks!